gtsam/gtsam.i

```diff
@@ -2759,21 +2759,35 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtsam/slam/dataset.h>
+// Dummy classes, for MATLAB wrappers
+class SfmMeasurement{};
+class SiftIndex{ };
+class SfmMeasurements{};
+class SfmCamera{};
```

Comment:
I dont think we need to wrap SfmCamera, this is being wrapped as PinholeCameraCal3Bundler in another PR, and there are a few methods in other classes that use it. Please refer to https://github.com/borglab/gtsam/pull/556. Perhaps we can merge that PR if it is needed for this code.